src/vs/platform/theme/common/themeService.ts

```diff
@@ -25,10 +25,11 @@ export function themeColorFromId(id: ColorIdentifier) {
 // theme icon
 export interface ThemeIcon {
 	readonly id: string;
+	readonly themeColor?: ThemeColor;
 }
 
 export namespace ThemeIcon {
-	export function isThemeIcon(obj: any): obj is ThemeIcon {
+	export function isThemeIcon(obj: any): obj is ThemeIcon | { id: string } {
```

Comment:
@aeschli I added the `| { id: string }` here because bulkEditTree.ts relies on `ThemeIcon` only having an `id`.